homeassistant/components/pvpc_hourly_pricing/sensor.py

```diff
@@ -127,6 +128,8 @@ async def async_setup_entry(
 class ElecPriceSensor(CoordinatorEntity[ElecPricesDataUpdateCoordinator], SensorEntity):
     """Class to hold the prices of electricity as a sensor."""
 
+    _attr_has_entity_name = True
+
     def __init__(
```

Comment:
I think now you can remove the `name: str` arg, and remove the retrieval of the `name = entry.data[CONF_NAME]` in `async_setup_entry` 🤟